CMakeLists.txt

```diff
@@ -229,6 +239,12 @@ if(JEDI_DRIVER)
   list(APPEND _ufs_defs_private JEDI_DRIVER=ON)
 endif()
 
+if(GOCART)
+  add_dependencies(ufs UFS)
+  list(APPEND _ufs_defs_private FRONT_GOCART=Aerosol_Cap)
+  list(APPEND _ufs_libs_public UFS)
```

Comment:
What is `UFS` here?

Reply:
Changed to `UFS_Aerosols` in commit a3f15c4cc9724c26963a398e771730f1eb96e6d2